src/pc/1.13/ChunkColumn.js

```diff
@@ -0,0 +1,270 @@
+const SmartBuffer = require('smart-buffer').SmartBuffer
+const Vec3 = require('vec3').Vec3
+const ChunkSection = require('./ChunkSection')
+const constants = require('./constants')
+const BitArray = require('./BitArray')
+const varInt = require('./varInt')
+const assert = require('assert')
+
+// wrap with func to provide version specific Block
+module.exports = (Block, mcData) => {
```

Comment:
Why version specific ? Does it work for any other version ?

Reply:
I doubt this ChunkColumn class works for other versions right now, that is why I send along the version specifics things I need to use here. maybe it could be generalized to tend for older versions but I don't feel like making it now.